scvi/inference/autotune.py

```diff
@@ -155,12 +160,28 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def configure_asynchronous_logging(logging_queue: multiprocessing.Queue):
+    """Helper for asynchronous logging - Writes all logs to a queue."""
+    root_logger = logging.getLogger()
+    root_logger.setLevel(logging.DEBUG)
```

Comment:
Why do you set the root logger level?

Reply:
This the root_logger of an auxiliary process which has its own python interpreter (spawn), all the logs are sent back to the main process using a handler in the root_logger, they are then dispatched to the according logger and handled as they normally would by the main process' python interpreter.